### DISCUSSION TITLE
Better Platform.

### DISCUSSION THREAD

**Discussion by komiljonmaksudov** [yilmaznaslan/berlin-auslaenderbehorde-termin-bot#49] [General]
Hi everyone.

That's great what u did and respectful. However, ur code solution is not great for everyone but people that know github and coding which only allows to run on localhost and potentially deploy to servers to run all the time. Accussing the private company for getting advantage ain't great as well(imho). They just took advantage of it.

I built a platform for this like a month ago and succeeded in it for non study/family visas by creating a fake account. However, it's been in beta since 3 weeks and haven't found any visa appointments for 10+ users. I included the Anmeldung as well which was successful after a day even though they provide an api but the provided email in the robots.txt doesn't exist. Bummer. I didn't use ur code solution btw.

App: https://app.mili-my.name/ 
Landing Page: https://glowing-brand-015819.framer.app/

Only way would be rn to email them which I included in the F&Q section.

**Comment by yilmaznaslan** [comment]
I didn't understand the point of your post, to be honest. What is the thing you are bringing to the discussion? Your website looks nice, good job. But are you trying to boost your paid web service through this open-source project, really :) ?

- Of course, this project is not great for everyone. It is in GitHub and it requires minimum coding knowledge
- I am and I will accuse such companies that charged 50€ for this non-sens. If you, me, or any other person start to deploy these codes 24x7 in the servers without any limitation, no one ever will be able get any termin, including bots as well.
- Nevertheless, the whole auslaenderbehrode system is not even working properly and not giving any termins at all, you are right, I think it really won't be worth it to invest more time&money&energy on this stupid system.

**Comment by komiljonmaksudov** [comment]
My bad... I was a bit in rush.

1. But are you trying to boost your paid web service through this open-source project, really :) ? 
Nope. I didn't mean it and I guess I'd open source it. Eventually, I wanted to activate the stripe which is in beta rn only for covering the server cost. Or I'll use it further as a portfolio project or repurpose for other private booking platforms.
2. Of course, this project is not great for everyone. It is in GitHub and it requires minimum coding knowledge. 
That's where I want to clearify. Ur team did an incredible job but it is the code. That's the main problem.
3. I am and I will accuse such companies that charged 50€ for this non-sens. If you, me, or any other person start to deploy these codes 24x7 in the servers without any limitation, no one ever will be able get any termin, including bots as well.
True! That's a bad part of it and it is better not to deal with it but the system won't change it. As I've heard about, they mostly only give appointments via email. Imho, I'd not consider it as a bot more like a hacking solution))) Kidding more like an automation application.

True. How can we improve it as international people? I know the government is slow but can we do something about it?

**Comment by komiljonmaksudov** [comment]
Hi Mr. @yilmaznaslan.

An update from me, they introduced a recaptcha in the slots page. That's a good move but it still doesn't solve the main problem.
